src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -312,6 +367,15 @@ describe('Default reporter', () => {
           testsNotFound: new Set(['bbb-bbb-bbb']),
         },
       },
+      {
+        description: 'Case with 2 passed results, of which 1 comes from previous CI run',
+        summary: {
+          ...baseSummary,
+          expected: 1,
```

Comment:
I don't like this - I would still say that `2` results were "expected" - from the standpoint that the user "expects" that many results when they run the CI job - the fact that we selectively remove jobs does not change the expectations of the user.